Source/Redemption/Private/BaseLocalizationWidget.cpp

```diff
@@ -0,0 +1,25 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+
+#include "BaseLocalizationWidget.h"
+#include "Internationalization/Internationalization.h"
+#include "Internationalization/Culture.h"
+#include "Components/TextBlock.h"
+#include "Engine/Font.h"
+#include "LocalizationGameMode.h"
+#include "Kismet/GameplayStatics.h"
+#include "FontManager.h"
+
+
+void UBaseLocalizationWidget::NativeConstruct()
+{
+	Super::NativeConstruct();
+}
+
+UFont* UBaseLocalizationWidget::SetFontPerLanguage() const
+{
+	FString CurrentLanguage = FInternationalization::Get().GetCurrentCulture()->GetName();
```

Comment:
This is the sole way of changing the language correct?

Reply:
this is to get the current language, not change it.